alvr/client_openxr/src/extra_extensions/body_tracking_bd.rs

```diff
@@ -120,12 +122,76 @@ type GetBodyTrackingStateBD = unsafe extern "system" fn(
     *mut BodyTrackingErrorCodeBD,
 ) -> sys::Result;
 
+#[repr(C)]
+struct MotionTrackerConnectStateBD {
+    tracker_count: i32,
+    serials: [MotionTrackerSerialBD; 6],
+}
+
+#[repr(C)]
+#[derive(Copy)]
+pub struct MotionTrackerSerialBD {
+    pub serial: [u8; 24],
+}
+impl Clone for MotionTrackerSerialBD {
+    fn clone(&self) -> Self {
+        *self
+    }
+}
+
+#[repr(transparent)]
+#[derive(Copy, Clone, Eq, PartialEq)]
+pub struct MotionTrackerConfidenceBD(i32);
+
+#[repr(C)]
+pub struct MotionTrackerLocationBD {
+    pub pose: sys::Posef,
+    pub angular_velocity: sys::Vector3f,
+    pub linear_velocity: sys::Vector3f,
+    pub angular_acceleration: sys::Vector3f,
+    pub linear_acceleration: sys::Vector3f,
+}
+
+#[repr(C)]
+pub struct MotionTrackerLocationsBD {
+    pub serial: MotionTrackerSerialBD,
+    pub local_pose: MotionTrackerLocationBD,
+    pub confidence: MotionTrackerConfidenceBD,
+    pub global_pose: MotionTrackerLocationBD,
+}
+
+type GetMotionTrackerConnectStateBD =
+    unsafe extern "system" fn(sys::Instance, *mut MotionTrackerConnectStateBD) -> sys::Result;
+
+type GetMotionTrackerLocationsBD = unsafe extern "system" fn(
+    sys::Instance,
+    sys::Time,
+    *const MotionTrackerSerialBD,
+    *mut MotionTrackerLocationsBD,
+) -> sys::Result;
+
+type SetConfigPICO = unsafe extern "system" fn(sys::Session, i32, *const c_char) -> sys::Result;
+
+#[repr(transparent)]
+#[derive(Copy, Clone, Eq, PartialEq)]
+pub struct MotionTrackerModeBD(i32);
+impl MotionTrackerModeBD {
+    pub const BODY_TRACKING: MotionTrackerModeBD = Self(0i32);
+    pub const MOTION_TRACKING: MotionTrackerModeBD = Self(1i32);
+}
+
+type GetMotionTrackerModeBD =
+    unsafe extern "system" fn(sys::Instance, *mut MotionTrackerModeBD) -> sys::Result;
+
 pub struct BodyTrackerBD {
     handle: XrBodyTrackerBD,
     instance: sys::Instance,
```

Comment:
I'm noticing this now, instead of a instance raw handle, store a `xr::Session`. This ensures safety when the session/instance get destroyed.